ieee_tec_bot/ieee_tec_bot.py

```diff
@@ -281,9 +281,10 @@ def informationHandler(bot, update):
 
         elif update.message.text in customKeyboards[guideScreen][1]:
             #If Tech Chapters benefits selected, get the info from the info module
-            #replyText=info.chapterMembershipSteps()
-            replyText ="Mostrando pasos para pertenecer a un capítulo."
-            goToScreen(bot, update, message=replyText)
+            replyText=info.chapterMembershipSteps()
+            chapterMembershipDocument= open("MembresíaIEEE.pdf","rb")
```

Comment:
This won't work since the file is not in the same folder.
Try with:
```
membershipDocument= open("../docs/MembresiaIEEE.pdf","rb")
```
But since the file name can be modified I suggest creating a variable in the "config.py" file with the relative path. So the call will be more like:
```
membershipDocument= open(config.membershipPath,"rb")
```